### ISSUE TITLE
config/attributes not returning correctly

### ISSUE THREAD

**Issue by b3n4kh** [taranis-ai/taranis-ai#5]
The whole concept of Attributes vs AttributeEnum has to be revisited, there is potential for massive simplification.

**Comment by b3n4kh** [comment]
here I have a question: On line [397](https://github.com/ait-cs-IaaS/Taranis-NG/pull/162/files#diff-813203484a9ea4803feaa98ac3535abbc878d59d59757c435977a4fc6c175602R397), is it an expected way to use /attributes/id, or should the request be sent to /attributes?search= --> because that search seems to use the `type` key (of the JSON). But for /attributes?search=ENUM it returns inconsistently.
Additionally, the entries created by POST, like:
```
{
        "id": 44,
        "name": "TEST Attribute name",
        "description": "Simple attribute desc TEST",
        "type": "STRING"
}
```
are actually unsearchable using the ?search=STRING.
That's actually the reason why I access it using the ID directly.

**Comment by b3n4kh** [comment]
Fixed with: #267
